contracts/erc2o/include/erc2o/eosio.token.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <eosio/asset.hpp>
+#include <eosio/eosio.hpp>
+#include <string>
+
+namespace eosio {
+
+using std::string;
+
+class [[eosio::contract("eosio.token")]] token : public contract {
+   public:
+    using contract::contract;
+
+    [[eosio::action]] void create(const name& issuer,
+                                  const asset& maximum_supply);
+
+    [[eosio::action]] void issue(const name& to, const asset& quantity, const string& memo);
+
+    [[eosio::action]] void retire(const asset& quantity, const string& memo);
+
+    [[eosio::action]] void transfer(const name& from,
+                                    const name& to,
+                                    const asset& quantity,
+                                    const string& memo);
+
+    // exists to send along a std::basic_string<uint8_t> as a memo without copying over to string first
+    [[eosio::action]] void transferb(const name& from,
```

Comment:
This action is not part of the `eosio.token` compatible contracts that are typically deployed on EOS and so should not be included as part of this interface file.

Reply:
Also, if all you need is the `transfer` action, you could just keep that one and remove all the other ones you do not actually use to send as inline actions or receive as notifications.

Reply:
I just copied this one from evm runtime tests... is there a better place to get this file or deleting transferb will make it look good enough?

Reply:
A better place to get it from is https://github.com/eosnetworkfoundation/eos-system-contracts/blob/main/contracts/eosio.token/include/eosio.token/eosio.token.hpp. But again, I don't think it is worth having the entire thing. I think we only care about keeping the part of the interface we actually depend on. So all the private members can be removed and all the actions other than `transfer` (and its `action_wrapper`) could be removed.

Reply:
fixed